### DISCUSSION TITLE
how update?

### DISCUSSION THREAD

**Discussion by tynadra90** [5axes/Calibration-Shapes#64] [General]
Cura 4.11 has 1.3.3 ver. calibration shapes
![Screenshot_1](https://user-images.githubusercontent.com/26870911/137451546-ec826644-adea-42b9-93e6-2a693f02f626.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/26870911/137451547-e89a429b-1f10-4d02-bd44-90b3411f8c39.jpg)

**Comment by 5axes** [comment]
That's normal it's the last release in the marketplace . No need to update for the moment

**Reply by tynadra90** [reply]
how add Tower Flow Test? thx

**Reply by 5axes** [reply]
Right now, I'm not really satisfied of this test. Need to be modified or to fine a better design.
